lldb/packages/Python/lldbsuite/test/dotest_args.py

```diff
@@ -164,6 +164,11 @@ def create_parser():
         dest='clang_module_cache_dir',
         metavar='The clang module cache directory used by Clang',
         help='The clang module cache directory used in the Make files by Clang while building tests. Defaults to <test build directory>/module-cache-clang.')
+    group.add_argument(
+        '--lldb-swift-libs',
```

Comment:
About the name of this option: What is "lldb" about it? Should this be `--swift-libs-dir`?

Reply:
The name is what the CMake variable is called, I'm just keeping it the same.

Reply:
I would still vote for giving this option a more descriptive name.